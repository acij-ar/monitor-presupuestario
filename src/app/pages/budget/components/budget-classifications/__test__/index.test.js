const React = require('react');
const { shallow } = require('enzyme');

const BudgetClassifications = require('..');

describe('BudgetClassifications component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<BudgetClassifications/>);
    expect(wrapper).toMatchSnapshot();
  });
});
