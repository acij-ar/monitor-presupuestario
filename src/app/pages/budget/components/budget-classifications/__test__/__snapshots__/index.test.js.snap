// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudgetClassifications component should match snapshot 1`] = `
<ContentSection
  id="budget-classifications"
  title="Clasificaciones"
>
  <p>
    La información presupuestaria contenida en los distintos documentos puede organizarse y clasificarse de muchas formas, que serán más o menos útiles según el análisis que se quiera hacer. Las clasificaciones presupuestarias son formas de agrupar u ordenar los recursos y gastos según diferentes criterios, que responden algunas preguntas básicas: ¿Quién gasta?, ¿En qué se gasta?, ¿Para qué se gasta?, o ¿Dónde se gasta?.
  </p>
  <p>
    El detalle de las clasificaciones se ve en las distintas planillas o tablas, que nos permitirán encontrar con facilidad la información relevante para nuestro análisis. A continuación presentamos las clasificaciones más útiles para conocer rápidamente el presupuesto:
  </p>
</ContentSection>
`;
